src/AspNet.Security.OAuth.Introspection/Events/AccessTokenReceivedContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace AspNet.Security.OAuth.Introspection {
+    public class AccessTokenReceivedContext : BaseIntrospectionContext {
+        public AccessTokenReceivedContext(
+            HttpContext context, 
```

Comment:
Ah. I thought this was akin to asserting that something isn't null and makes it so you can't pass in null at all. If it doesn't, well, that's ok. Still a nice-to-have.